KerbalStuff/blueprints/anonymous.py

```diff
@@ -59,27 +51,26 @@ def content(path: str) -> werkzeug.wrappers.Response:
 
 @anonymous.route("/browse")
 def browse() -> str:
-    featured = Featured.query.order_by(desc(Featured.created)).limit(6).all()
-    top = search_mods(None, '', 1, 6)[0]
-    new = Mod.query.filter(Mod.published).order_by(desc(Mod.created)).limit(6).all()
+    featured = get_featured_mods(None, 6)
+    top = get_top_mods(None, 6)
+    new = get_new_mods(None, 6)
     return render_template("browse.html", featured=featured, top=top, new=new)
 
 
 @anonymous.route("/browse/new")
 def browse_new() -> str:
     mods = Mod.query.filter(Mod.published).order_by(desc(Mod.created))
```

Comment:
Could this use `get_new_mods`?

Reply:
I don't think so, since we're sending it through `paginate_query()`  afterwards, which makes sure we only get a certain amount of mods, from a certain offset based on `request.args.get('page', '')`.

But we could call `get_page()` in all the routes, send it as argument to the `get_*_mods()` functions, and incorporate the paging logic into them, and get rid of `paginate_query()`.
It would introduce slight code duplication, but it would be in one place, and potentially bumping performance by a bunch.
Should I add that to this PR?

Reply:
Ahh, I see now. No, don't add to this PR, it's fine as is.
I saw "new" and thought you just missed a line, but right, we can browse deeper into this list so `get_new_mods` won't work.